string type_name
float64[] dimensions
Transform2D offset
